test.sh

```diff
@@ -0,0 +1,14 @@
+APP_NAME=test
```

Comment:
You want to include this test.sh and test.out files?

Reply:
No, that was an accident. I'll remove them